udata/features/territories/commands.py

```diff
@@ -71,8 +71,13 @@ def collect_references_files():
                     continue
                 filename = resource.url.split('/')[-1]
                 reference_path = references.path(filename)
+                log.info('Found reference: {reference_path}'.format(
+                    reference_path=reference_path))
```

Comment:
Or you can use the logging extrapolation:
```python
log.info('Found reference: %s', reference_path)
```
Extrapolation is only evaluated on display